src/TestInfrastructure/ToolWindowCommandTests.cs

```diff
@@ -0,0 +1,137 @@
+﻿/*
+ * SonarLint for Visual Studio
+ * Copyright (C) 2016-2020 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using System;
+using FluentAssertions;
+using Microsoft.VisualStudio.Shell;
+using Microsoft.VisualStudio.Shell.Interop;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Moq;
+using Moq.Protected;
+
+namespace SonarLint.VisualStudio.Integration.UnitTests
+{
+    public abstract class ToolWindowCommandTests<T> where T : ToolWindowPane
```

Comment:
This used to be in the IssueViz.UnitTests.
I'm a bit on the fence if this test suite should even exist.

Reply:
Why? Is it because you think none of the tests are useful?

Reply:
I think it adds complexity to the tests. The alternative is repetition but in a test context it might be more readable and easier to maintain. I'll keep the suite for now but I'm on the fence about its usefulness in the long run.